en/peripherals/gripper_servo.md

```diff
@@ -0,0 +1,35 @@
+# Servo Gripper with PWM
+
+This section describes how to connect and configure a Servo type Gripper with PWM input.
+
+![High-load gripper example](../../assets/hardware/grippers/highload_gripper_example.jpg)
+
```

Comment:
Do we know the model of this gripper - could we have a list here of supported grippers, with this as the first one?

Reply:
Ah sure, I will ask!

Reply:
Hmm, it seems like it is just a generic gripper from a company named 'Southco', and it's not really a well known product. Would it be beneficial to include it regardless? (Concern is that it doesn't represent the off the shelf gripper that well)

Reply:
Yes I think so, though perhaps with a disclaimer that this is a tested servo, not necessarily a recommendation.

Reply:
Finally found it! https://southco.com/en_any_int/r4-em-r22-161

Reply:
Thanks !!! I moved this note into a "Supported Grippers" section in the servo. This allows us to easily separate the mavlink and other grippers into particular types, depending on what people want to do.